.github/workflows/linter.yml

```
@@ -0,0 +1,65 @@
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+# For the Web Almanac we have the following rule changes from the defaults:
+#  * Markdown - we're a bit more relaxed about this since authors may be less technical
+#  * Python - Only change from default is to set ignored-modules=flask, flask-talisman...etc.
+#  * CSS - Tweaked to ignore some Prince/Normalise exceptions and some whitespace exceptions
+#  * JS - default rules
+#  * JSON - default rules
+#  * YML - default rules
+# Exception config files are in the .github/linters directory
+
+######################################
+# Start the job on all pull requests #
+######################################
+on:
+  - push
+  - pull_request
+
+###############
+# Set the Job #
+###############
+jobs:
+  build:
```

It is a not a build job (not that it will affect the process).

```suggestion
  lint:
```

---

Fixed.